src/custom/utils/trade.ts

```diff
@@ -67,24 +67,18 @@ function _getSummary(params: PostOrderParams): string {
   }
 }
 
-export async function signAndPostOrder(params: PostOrderParams): Promise<AddUnserialisedPendingOrderParams> {
-  const {
-    kind,
-    chainId,
-    inputAmount,
-    outputAmount,
-    sellToken,
-    buyToken,
-    feeAmount,
-    validTo,
-    account,
-    signer,
-    recipient,
-    allowsOffchainSigning,
-    appDataHash,
-    sellAmountBeforeFee,
-    quoteId,
-  } = params
+export function getOrderParams(params: PostOrderParams): {
+  summary: string
+  quoteId: number | undefined
+  order: UnsignedOrder
+} {
+  const { kind, inputAmount, outputAmount, sellToken, buyToken, feeAmount, validTo, recipient, appDataHash, quoteId } =
+    params
+  const sellTokenAddress = sellToken instanceof Token ? sellToken.address : NATIVE_CURRENCY_BUY_ADDRESS
```

Comment:
It's a wrong check, we also have WrappedTokenInfo. So, it would be better:
```
const sellTokenAddress = sellToken.isNative ? NATIVE_CURRENCY_BUY_ADDRESS : sellToken.address
```

Or just use `sellToken.address`

Reply:
If I just use `sellToken.address`, that won't have the native currency address (0xeee...), isn't it?

Reply:
https://github.com/cowprotocol/cowswap/pull/1377